### PR TITLE
Fix/qa

### PR DESCRIPTION
**Pull request by fedgut**
## Quick Info
Fix most QA observations

## What does this change?
- Fix styles
- Implement email confirmation

## Why are you changing that?
To have a better TIL

## How do you manually test this?
create an account! 

## Screenshots

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -43,13 +43,16 @@ def to_param
   end
 
   def increment_likes
+    return if current_developer == post.developer
```
Comment (mumoc) [inline app/models/post.rb]: you can move this to a method,  like:

```
def check_autor
  return if current_developer == post.developer
end
```

and use it as a before_action
```
before_action :check_developer, only: [:increment_likes, :decrement_likes]
```

**Review by mumoc — CHANGES_REQUESTED** [review]
Just one small thing

**Review by vicmaster — COMMENTED** [review]
What do you think about using micro-commits instead of grouping a set of fixes in one commit to separate them into descriptive fixes instead of `QA fixes`?.

It would help us to understand the `why` on these subsets of updates/changes.

For example:

Commit 1 - `Update alert messages background color.`

Changes from commit 1, just updating background color on Alert messages.

**Comment by fedgut** [comment]
> What do you think about using micro-commits instead of grouping a set of fixes in one commit to separate them into descriptive fixes instead of `QA fixes`?.
> 
> It would help us to understand the `why` on these subsets of updates/changes.
> 
> For example:
> 
> Commit 1 - `Update alert messages background color.`
> 
> Changes from commit 1, just updating background color on Alert messages.



> What do you think about using micro-commits instead of grouping a set of fixes in one commit to separate them into descriptive fixes instead of `QA fixes`?.
> 
> It would help us to understand the `why` on these subsets of updates/changes.
> 
> For example:
> 
> Commit 1 - `Update alert messages background color.`
> 
> Changes from commit 1, just updating background color on Alert messages.

Absolutly right @vicmaster! Will do it like that in the future.

File: app/models/post.rb (hunk 1 above)
Comment (fedgut) [reply app/models/post.rb]: Indeed! I'll keep DRY in mind

**Pull request merged**